src/einsteinpy/symbolic/tensor.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, arr, config="ll"):
 
         if isinstance(arr, (list, tuple, np.ndarray)):
             self.arr = sympy.Array(arr)
+        elif isinstance(arr, (int, float)):
```

Comment:
Scalars can be sympy expressions also. You need to include that case.
```py
from sympy.core.expr import Expr
# blah-blah codes in between
# check through isinstance
```

Reply:
no need for an `elif`
you could have done the same thing in the above `isinstance`.
Like this `if isinstance(arr, (list, tuple, np.ndarray, int, float, Expr)):`